### DISCUSSION TITLE
A few suggestions

### DISCUSSION THREAD

**Discussion by svefro** [tomasklaen/uosc#60] [Ideas]
-pressing left when browsing files in the menu should take me to the parent directory
-navigating to different drives should be possible on windows

-Dragging the window to the top/left or right should snap the window like all other windows can in windows
-Maximize button should maximize window not go fullscreen

**Comment by tomasklaen** [comment]
> pressing left when browsing files in the menu should take me to the parent directory
> navigating to different drives should be possible on windows

Yes that would be nice. If I find some time to work on uosc I'll add this, but I have a lot of other obligations atm, can't even give you ETA.

> Dragging the window to the top/left or right should snap the window like all other windows can in windows

That's mpv configuration. Add `snap-window=yes` to `mpv.conf`

> Maximize button should maximize window not go fullscreen

I agree, that's why uosc is just using mpv command `cycle window-maximized`. Dunno why it fullscreens instead. If there is a more appropriate command to use here, let me know. I'm not aware of it -.-

**Comment by svefro** [comment]
No ETA needed.

There is one more thing i have noticed that seams a bit wierd.
When scrolling down on the video it seeks backwards witch is a bit counter intuitive.
When hoovering over the progress bar at the bottom it seeks forward when scrolling down.

i think scrolling down should seek forward both places.

**Reply by svefro** [reply]
nevermind.
i changed seek direction in input.conf by adding
WHEEL_UP seek -10
WHEEL_DOWN seek 10

**Reply by Neo-XP** [reply]
I wanted the opposite on the timeline (go forward with "WHEEL_UP" and backward with "WHEEL_DOWN"), so I changed this in the uosc.lua script:

```
	on_wheel_up = function(this)
		if options.timeline_step > 0 then mp.commandv('seek', -options.timeline_step) end
	end,
	on_wheel_down = function(this)
		if options.timeline_step > 0 then mp.commandv('seek', options.timeline_step) end
	end,
```

By this:

```
	on_wheel_up = function(this)
		if options.timeline_step > 0 then mp.commandv('seek', options.timeline_step) end
	end,
	on_wheel_down = function(this)
		if options.timeline_step > 0 then mp.commandv('seek', -options.timeline_step) end
	end,
```

But it would be nice to make it an option in the uosc.conf file, so one does not have to change this with every new version of the file.

I tried to change "timeline_step=5" by "timeline_step=-5" in the uosc.conf file, but it did not work.

**Comment by svefro** [comment]
I have noticed one more thing 

When opening the playlist view the scroll pos always starts at the top.

I think it should scroll to active item in center or use the last scroll position if the playlist has not changed since last open.
